db/migrate/20220330211042_create_users.rb

```diff
@@ -3,7 +3,7 @@ def change
     create_table :users do |t|
         # t.integer   :identifier, auto_increment: true, null: false
         t.string    :username
-        t.string    :password
+        t.string    :password_digest
```

Comment:
What exactly is the difference in the change?

Reply:
This is for the bcrypt gem. While the rest of the app refers to the user's password as just `password`, to securely save it in the database the bcrypt gem calls it password_digest. I'm not entirely sure what it does but it makes the login w/ bcrypt work.

Reply:
epic-- thank you!